docs/iris/src/whatsnew/1.10.rst

```diff
@@ -0,0 +1,170 @@
+
+Deprecations
+============
+* Deprecated the :class:`iris.Future` option
+  ``iris.FUTURE.strict_grib_load``.
+  This only affected the module :mod:`iris.fileformats.grib`, which is itself
+  now deprecated.
+  Please see :ref:`iris_grib package <iris_grib_added>`, above.
+* Deprecated the module :mod:`iris.fileformats.grib`.  The new package
+  `iris_grib <https://github.com/SciTools/iris-grib>`_ replaces this
+  fuctionality, which will shortly be removed.
+  Please see :ref:`iris_grib package <iris_grib_added>`, above.
+* The use of :data:`iris.config.SAMPLE_DATA_DIR` has been deprecated and replaced by the now importable `iris_sample_data <https://github.com/SciTools/iris-sample-data>`_ package.
+ 
+* Deprecated the module :mod:`iris.analysis.interpolate`.
+  This contains the following public items, all of which are now deprecated and
+  will be removed in a future release:
+
+    * :func:`~iris.analysis.interpolate.linear`
+    * :func:`~iris.analysis.interpolate.regrid`
+    * :func:`~iris.analysis.interpolate.regrid_to_max_resolution`
+    * :func:`~iris.analysis.interpolate.nearest_neighbour_indices`
+    * :func:`~iris.analysis.interpolate.nearest_neighbour_data_value`
+    * :func:`~iris.analysis.interpolate.extract_nearest_neighbour`
+    * class :class:`~iris.analysis.interpolate.Linear1dExtrapolator`.
+
+  Please use the replacement facilities individually noted in the module
+  documentation for :mod:`iris.analysis.interpolate`
+* The method :meth:`iris.cube.Cube.regridded` has been deprecated.
+  Please use :meth:`iris.cube.Cube.regrid` instead (see
+  :meth:`~iris.cube.Cube.regridded` for details).
+* Deprecated :data:`iris.fileformats.grib.hindcast_workaround` and :class:`iris.fileformats.grib.GribWrapper`. The class :class:`iris.fileformats.grib.message.GribMessage` provides alternative means of working with GRIB message instances.
+* Deprecated the module :mod:`iris.fileformats.ff`.  Please use the replacement
+  facilities in module :mod:`iris.fileformats.um` :
+
+  * :func:`iris.fileformats.um.um_to_pp` replaces :class:`iris.fileformats.ff.FF2PP`.
+  * :func:`iris.fileformats.um.load_cubes` replaces :func:`iris.fileformats.ff.load_cubes`.
+  * :func:`iris.fileformats.um.load_cubes_32bit_ieee` replaces :func:`iris.fileformats.ff.load_cubes_32bit_ieee`.
+
+  All other public components are generally deprecated and will be removed in a future release.
+* The :func:`iris.fileformats.pp.as_pairs` and :func:`iris.fileformats.grib.as_pairs` are deprecated.  These are replaced with :func:`iris.fileformats.pp.save_pairs_from_cube` and :func:`iris.fileformats.grib.save_pairs_from_cube`.
+* ``iris.fileformats.pp_packing`` has been hidden behind a deprecation interface. Please install `mo_pack <https://github.com/SciTools/mo_pack>`_ to use instead.
```

Comment:
Could be clearer I think.
Suggest :
"* ``iris.fileformats.pp_packing`` is now deprecated :  Please install the separate package `mo_pack <https://github.com/SciTools/mo_pack>`_ instead.  This provides the same functionality."